sos/report/plugins/salt.py

```diff
@@ -26,12 +27,38 @@ def setup(self):
         else:
             self.add_copy_spec("/var/log/salt")
 
-        self.add_copy_spec("/etc/salt")
-        self.add_forbidden_path("/etc/salt/pki/*/*.pem")
+        self.add_copy_spec(["/var/log/venv-salt-minion.log",
+                            "/var/log/salt-ssh.log"])
+
+        self.add_copy_spec(["/etc/salt",
+                            "/etc/venv-salt-minion/",
+                            "/usr/local/etc/salt"])
+        self.add_forbidden_path(["/etc/salt/pki/*/*.pem",
+                                 "/etc/venv-salt-minion/pki/*/*.pem",
+                                 "/usr/local/etc/salt/pki/*/*.pem"])
+
+        self.add_copy_spec(["/etc/machine-id",
+                            "/var/lib/dbus/machine-id"])
```

Comment:
Style nit - sos style is to indent from the caller for list elements and close the bracket on the ending line, e.g.:

```python
self.add_copy_spec([
    '/path/one',
    '/path/two',
])
```

Reply:
Unified all lists with this indentation style.